literals/rand.go

```diff
@@ -0,0 +1,34 @@
+package literals
+
+import (
+	"crypto/rand"
+	"fmt"
+	mathrand "math/rand"
+	"os"
+	"strings"
+)
+
+var envGarbleSeed = os.Getenv("GARBLE_SEED")
+
+// If math/rand.Seed() is not called, the generator behaves as if seeded by rand.Seed(1),
+// so the generator is deterministic.
+
+// genRandBytes return a random []byte with the length of size.
+func genRandBytes(size int) []byte {
```

Comment:
a better signature is `randBytes(dst []byte)`, because:

1) you only need one parameter, the size is `len(dst)`
2) the caller can choose how to allocate the space, e.g. in the stack
3) the caller can reuse a single buffer between many calls